vulnerable_people_form/form_pages/contact_details.py

```diff
@@ -1,13 +1,20 @@
+import logging
+
 import phonenumbers
 from flask import redirect, session
 
 from .blueprint import form
+from .shared.logger_utils import init_logger
 from .shared.render import render_template_with_title
 from .shared.routing import route_to_next_form_page, get_back_url_for_contact_details
 from .shared.session import form_answers, get_errors_from_session, request_form
 from .shared.validation import validate_contact_details
 
 
+logger = logging.getLogger(__name__)
```

Comment:
this logger is never used in this file?

Reply:
`logger` is used on line 43. `if not validate_contact_details("check_contact_details", logger=logger):`

To fit existing current architecture I have instantiated loggers in the view files. They get passed down to the validation code as parameters